libs/strings/string.go

```diff
@@ -55,6 +55,10 @@ func SplitAndTrim(s, sep, cutset string) []string {
 	return spl
 }
 
+// TrimWhitespace removes all leading and trailing whitespace from the
+// string.
+func TrimWhitespace(s string) string { return strings.Trim(s, "\t\n ") }
```

Comment:
Could we use [`strings.TrimSpace`](https://godoc.org/strings#TrimSpace) instead?